web/source/kmwembedded.ts

```diff
@@ -531,7 +525,7 @@ namespace com.keyman.text {
     try {
       // Now that we've manually constructed a proper keystroke-sourced KeyEvent, pass control
       // off to the processor for its actual execution.
-      return keyman.core.processKeyEvent(Lkc) != null;
+      return !keyman.core.processKeyEvent(Lkc);
```

Comment:
If the original check was for something to not be `null`, you want something truthy.  A single `!` coerces to `false`, so use a double (`!!`) to convert something truthy to `true`.

```suggestion
      return !!keyman.core.processKeyEvent(Lkc);
```

Reply:
I'm guessing what you actually got was `undefined`?  A quick console check does verify that `undefined != null` results in `false`.

That said... I should probably check and see where the `undefined` is coming from.  That sounds unintended.

Reply:
Looking at the method... it looks like its return types never got fully reworked.  The intent was for the method to return either a `RuleBehavior` or `null` for any case, but there are some `undefined` and `true` cases remaining there.  I should probably address that once my current predictive tasks are done.